roles.json

```
@@ -10,21 +10,33 @@
             "Erik Tollerud"
         ],
         "role-head": "Coordination committee member",
-        "responsibilities": {
-            "description": "Overall coordination and management of the Astropy project, including:",
-            "details": [
-		"Formally detailed in the <a href='https://github.com/astropy/astropy-APEs/blob/main/APE0.rst#the-coordination-committee'>The Astropy Project Governance Charter (APE 0)</a>",
-                "Keeping a large-scale view of the Astropy ecosystem",
-                "Evaluating new affiliated package submissions and review of existing affiliated packages",
-                "Approving or rejecting Astropy APEs",
-                "Evaluating and merging core package pull requests as needed (e.g., for sub-packages without a maintainer)",
-                "Arbitrating disagreements in the core package, including final decisions when otherwise deadlocked",
-                "Maintaining the list of roles and related github permissions",
-                "Managing finances for the project",
-                "Coordinating with NumFOCUS and other funding organizations",
-                "Securing funding for the project via discussions and proposals to funding agencies"
-            ]
-        }
+        "responsibilities": [
+            {
+                "description": "Overall coordination and management of the Astropy project, including:",
+                "details": [
+		            "Formally detailed in the <a href='https://github.com/astropy/astropy-APEs/blob/main/APE0.rst#the-coordination-committee'>The Astropy Project Governance Charter (APE 0)</a>",
+                  "Keeping a large-scale view of the Astropy ecosystem",
+                  "Evaluating new affiliated package submissions and review of existing affiliated packages",
+                  "Approving or rejecting Astropy APEs",
+                  "Evaluating and merging core package pull requests as needed (e.g., for sub-packages without a maintainer)",
+                   "Arbitrating disagreements in the core package, including final decisions when otherwise deadlocked",
+                   "Maintaining the list of roles and related github permissions",
+                   "Managing finances for the project",
+                   "Coordinating with NumFOCUS and other funding organizations",
+                   "Securing funding for the project via discussions and proposals to funding agencies"
+                 ]
+            },
+            {
+                "description": "Organisation and communication:",
```

```suggestion
                "description": "Organization and communication:",
```
For consistency with the rest of the page.